docker-compose.yml

```diff
@@ -14,14 +14,26 @@ services:
       - SPRING_DATASOURCE_URL=jdbc:postgresql://bar-database:5432/bar
       - SPRING_DATASOURCE_USERNAME=bar
       - SPRING_DATASOURCE_PASSWORD=bar
-      - AUTH_IDAM_CLIENT_BASEURL=https://idam-test.dev.ccidam.reform.hmcts.net
-      - PAYMENT_API_URL=http://payment-api-aat.service.core-compute-aat.internal
-      - S2S_AUTH_URL=http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
-      - S2S_SECRET=xxx
+      - auth.idam.client.baseUrl=http://idam-api:5000
+      - payment.api.url=http://payment-api-aat.service.core-compute-aat.internal
```

Comment:
We should have payment API instance running in bar API docker-compose, instead of pointing to payment API AAT. And hence please do the required changes.

Reply:
payment docker is added